Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -100,7 +115,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $purger = new ORMPurger($em);
         $purger->setPurgeMode($input->getOption('purge-with-truncate') ? ORMPurger::PURGE_MODE_TRUNCATE : ORMPurger::PURGE_MODE_DELETE);
         $executor = new ORMExecutor($em, $purger);
-        $executor->setLogger(function($message) use ($output) {
+        $executor->setLogger(function ($message) use ($output) {
```

Comment:
phpcs reported this line. https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#6-closures